src/misc/lv_cache.c

```diff
@@ -42,6 +42,25 @@ void _lv_cache_init(void)
     lv_mutex_init(&_cache_manager.mutex);
 }
 
+void lv_cache_replace_manager(lv_cache_manager_t * new_cache_manager)
```

Comment:
In order to use less keywords I suggest using 
```suggestion
void lv_cache_set_manager(lv_cache_manager_t * new_cache_manager)
```

What do you think?

Reply:
Good idea.
Renamed.